src/main/java/homeWorkPart1/tasks2and3/Main.java

```diff
@@ -1,4 +1,4 @@
-package Tasks2and3;
+package homeWorkPart1.tasks2and3;
 
 public class Main {
 
```

Comment:
to follow polymorphism principle, you should store your `circle` and `point` object in a `Movable` instance rather than `MovableCircle` or `MovablePoint` since they are already implementing the `Movable` interface